dscblog/paths.py

```diff
@@ -451,6 +454,7 @@ def blog_comment(request):
             else:
                 comment = request.user.comment(
                     blog=b, text=request.POST['text'].strip(), reference=ref)
+                Alert.create_alert(ref_user=request.user, type=Alert.COMMENT_REPLY if ref else Alert.COMMENT, blog=b)
```

Comment:
The user getting the alert for COMMENT_REPLY should be the ref comment's user (the person who the user replied to) not the blog author. This is causing some bugs like if i comment on someone's blog and the author replies me back im not getting the alert (as im not the author)